.github/workflows/test-and-build.yml

```diff
@@ -5,7 +5,7 @@ on: [pull_request, push]
 env:
   PLUGIN_NAME: dashboards-search-relevance
   OPENSEARCH_VERSION: "main"
```

Comment:
OpenSearch version still main here if plugin is 2.4.0.0?

Reply:
good catch. updating.